src/cursor.c

```diff
@@ -56,7 +64,7 @@ cursor_rebase(struct seat *seat, struct wlr_scene_node *node,
 		wlr_seat_pointer_notify_clear_focus(seat->seat);
 		wlr_seat_pointer_notify_enter(seat->seat, surface, sx, sy);
 		wlr_seat_pointer_notify_motion(seat->seat, time_msec, sx, sy);
-	} else {
+	} else if (focused_surface) {
 		cursor_set(seat, "left_ptr");
```

Comment:
It might be nice to use the `XCURSOR_DEFAULT` macro here, to be consistent with other call sites such as in `process_cursor_motion()`.

Reply:
changed both occurrences to `XCURSOR_DEFAULT`